VSDiagnostics/VSDiagnostics/VSDiagnostics/Diagnostics/General/EqualsAndGetHashcodeNotImplementedTogether/EqualsAndGetHashcodeNotImplementedTogetherAnalyzer.cs

```diff
@@ -39,18 +45,17 @@ private void AnalyzeNode(SyntaxNodeAnalysisContext context)
                 }
 
                 var methodDeclaration = (MethodDeclarationSyntax)node;
-
                 if (!IsOverride(methodDeclaration))
                 {
                     continue;
                 }
 
-                if (methodDeclaration.Identifier.ValueText == nameof(Equals) && methodDeclaration.ParameterList.Parameters.Count == 1)
+                if (context.SemanticModel.GetDeclaredSymbol(methodDeclaration).OverriddenMethod == objectEquals)
```

Comment:
This case does not work.  For the base class, the containing namespace of the parent Equals/GetHashCode is System.Object.  For the derived class, the parent namespace is the namespace X is defined in.  This causes the expected and actual Equals/GetHashCode methods to be different.